test/inspector/test-inspector-port-cluster.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const assert = require('assert');
+const cluster = require('cluster');
+const common = require('../common');
+const debuggerPort = common.PORT;
+const childProcess = require('child_process');
+
+common.skipIfInspectorDisabled();
+
+let offset = 0;
+
+/*
+ * This test suite checks that inspector port in cluster is incremented
+ * for different execArgv combinations
+ */
+
+function testRunnerMain() {
+  spawnMaster({
+    execArgv: ['--inspect'],
+    workers: [{expectedPort: 9230}]
+  });
+
+  let port = debuggerPort + offset++ * 10;
+
+  spawnMaster({
+    execArgv: [`--inspect=${port}`],
+    workers: [
```

Comment:
This could be fragile (because of infra issues) maybe try the argv parsing approach for the test like in https://github.com/nodejs/node/pull/13373
Ref: https://github.com/nodejs/node/issues/13343

Extra: when using ports in the ephemeral range (not 9229, but maybe 12346) we get port collisions when doing `port +[1,2,3]`